assets/css/langsel.css

```diff
@@ -0,0 +1,118 @@
+#langselect-container {
```

Comment:
Maybe we'd just put this in fw.css as we need to load this on every page.

Reply:
Done.